spec-main/chromium-spec.ts

```diff
@@ -1947,9 +1947,13 @@ describe('navigator.hid', () => {
       expect(grantedDevices).to.not.be.empty();
       const altFixturesPath = path.resolve(__dirname, 'fixtures');
       w.loadFile(path.join(altFixturesPath, 'blank.html'));
-      await emittedOnce(w.webContents, 'did-navigate');
-      const grantedDevicesOnNewPage = await w.webContents.executeJavaScript('navigator.hid.getDevices()');
-      expect(grantedDevicesOnNewPage).to.be.empty();
+      const [,,,,, frameProcessId, frameRoutingId] = await emittedOnce(w.webContents, 'did-frame-navigate');
+      const frame = webFrameMain.fromId(frameProcessId, frameRoutingId);
+      expect(frame).to.not.be.empty();
+      if (frame) {
```

Comment:
`frame` is always truthy because of the `expect` on the previous line, yes?

Reply:
yes.  But I had to add the `if (frame)` to make typescript happy because const frame = webFrameMain.fromId can be empty